uisdk/src/main/res/layout/uisdk_quotes_fragment.xml

```diff
@@ -10,6 +10,19 @@
     android:background="@color/kh_uisdk_background2"
     >
 
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="4dp">
+        <ProgressBar
+            android:id="@+id/progressBar"
+            android:layout_width="match_parent"
+            android:layout_height="16dp"
+            android:indeterminate="true"
+            android:layout_gravity="center"
+            style="@style/Widget.AppCompat.ProgressBar.Horizontal"
+            android:indeterminateTint="@color/kh_uisdk_secondary"/>
```

Comment:
create a style that has parent="Widget.AppCompat.ProgressBar.Horizontal" in styles.xml for this control and move everything from here in there, except for id to be customizable by DP and also use some dimension from @dimen/ instead of 16dp or 4dp